src/lib/RichText/Converter/ListLineBreaks.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\FieldTypeRichText\RichText\Converter;
+
+use EzSystems\EzPlatformRichText\eZ\RichText\Converter;
+use DOMDocument;
+use DOMXPath;
+
+/**
+ * Class ListLineBreaks.
+ *
+ * Processes lists below <literallayout> tag.
+ */
+class ListLineBreaks implements Converter
```

Comment:
Naming & PHPDoc & extensibility:
```suggestion
/**
 * @internal
 *
 * Processes lists nested in <literallayout> tags.
 */
final class LiteralLayoutNestedList implements Converter
```
Intentionally skipping LineBreak(s) as this is too much of an implementation detail. It's just a converter handling nested lists in `<literallayout>`.

_Side note: I would suffix the name with Converter just to distinguish it from potential ValueObject, but that's inconsistent with the other Converter and there are probably different opinions about that, so not insisting ;)_

Reply:
Fixed as suggested in 295b2127692c3b2b2aaa26c0c9310cb3c916a2f4